.ci/smoke-tests.groovy

```diff
@@ -48,9 +48,27 @@ pipeline {
       }
       steps {
         dir ("${BASE_DIR}") {
-          withGoEnv() {
+          withTestClusterEnv {
+            script {
+              def smokeTests = sh(returnStdout: true, script: 'make smoketest/discover').trim().split('\r?\n')
+              def smokeTestJobs = [:]
+              for (smokeTest in smokeTests) {
+                smokeTestJobs["Run smoke tests in ${smokeTest}"] = {
+                  stage("Run smoke tests in ${smokeTest}") {
+                    sh(label: 'Run smoke tests', script: "make smoketest/run TEST_DIR=${smokeTest}")
```

Comment:
Would it be possible to have parameter for the `SMOKETEST_VERSIONS` variable in the Jenkins job that defaults to `7.17,latest`? That way the smoke tests will be run for `7.17` and the latest version.

We could also split that into two jobs? But it may be premature.